File: problems/algorithms/string-hamming/js/response/hamming.print.js
import { hamming } from "./hamming.js";

// Hamming distance

// comparing the GGACG to GGTCG
let dna1 = "GGACG";
let dna2 = "GGTCG";
//            ˆ

console.log(hamming(dna1, dna2));
console.log(1);

// comparing the GGACGGATTCTG to AGGACGGATTCT
let dna1 = "GGACGGATTCTG";
let dna2 = "AGGACGGATTCT";
//          ˆ ˆˆˆ ˆˆ ˆˆˆ

console.log(hamming(dna1, dna2));
console.log(9);

File: src/pages/exercises/_string-hamming/_codes/ruby/response/print/hamming.print.rb
require_relative "../src/hamming"

# Hamming distance

# comparing the GGACG to GGTCG
dna1 = 'GGACG'
dna2 = 'GGTCG'

p hamming(dna1, dna2)
p 1

# comparing the GGACGGATTCTG to AGGACGGATTCT
dna1 = 'GGACGGATTCTG'
dna2 = 'AGGACGGATTCT'

p hamming(dna1, dna2)
p 9
 